.github/workflows/integration_test.yaml

```diff
@@ -7,24 +7,25 @@ jobs:
   integration-test-lxd:
     strategy:
       matrix:
-        series: [xenial, bionic, focal]
+        # ubuntu 18.04 is no longer supported and hence xenial cannot be tested.
+        series: [bionic, focal]
         module: [agent, basic, nrpe, website]
       fail-fast: false
     name: Integration tests (lxd)
-    # [2022-09-29] The integration tests have to run on Ubuntu 18.04 to be able to support Xenial
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Setup operator environment
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: lxd
+          charm-channel: 2.x/stable
         # [2022-09-29] Need to install Python 3.8 because that is what the tests run on and 18.04
         # comes with Python 3.6
       - uses: actions/setup-python@v4
```

Comment:
I think this step is not needed anymore. Or at least, that's my take from the comment above

Reply:
The tests [require Python 3.8](https://github.com/yanksyoon/jenkins-charm/actions/runs/4662970163/jobs/8253901479) which isn't available without the setup step for Python.

Reply:
22.04 comes with python 3.10! :)

Reply:
Erf, so be it.